### PR TITLE
Fix the get_send_progress binding

### PR DESCRIPTION
**Pull request by asomers**

### CODE REVIEW

**Review by sonicaj — COMMENTED** [review]
@asomers we have a related PR which fixed the `zfs_ioctl.h` issue ( patch will be merged in master this week hopefully ). Can you please drop the changes made to `configure.ac` please ? Also can you please confirm if you have tested the patch yourself for `get_send_progress` ?

**Comment by asomers** [comment]
@sonicaj I'll rebase after you merge the other issue.  And I can confirm that I tested `get_send_progress`.

**Comment by sonicaj** [comment]
Sweet, thank you. We can move ahead once the other PR is merged then.
> And I can confirm that I tested `get_send_progress`.

**Comment by sonicaj** [comment]
@asomers can you please rebase and update ? Thank you

**Comment by asomers** [comment]
rebased

File: configure.ac [hunk 1]
```diff
@@ -101,6 +101,7 @@ else
 -I${freebsd_src}/cddl/compat/opensolaris/lib/libumem
 -I${freebsd_src}/cddl/contrib/opensolaris/lib/libzpool/common
 -I${freebsd_src}/sys/cddl/contrib/opensolaris/common/zfs
+-I${freebsd_src}/sys/cddl/contrib/opensolaris/uts/common/fs/zfs/sys
```
Comment (sonicaj) [inline configure.ac]: Is this still necessary after the `zfs_ioctl.h` fix in master ? I think not ? Can you please confirm ?

Comment (asomers) [reply configure.ac]: You're right.  I'll remove it.

**Review by sonicaj — APPROVED** [review]
Thank you

**Pull request merged**